recipes/inference_labels.ipynb

```diff
@@ -4,66 +4,88 @@
    "cell_type": "markdown",
```

Comment:
<p>there is no reason to have code formatting on:</p><pre class="ql-syntax" spellcheck="false">- Low inference scores can indicate either a model performing badly on some kind of data, or a disagreement between humans and the model. This can help you to: 
    - <code>Detect data drift</code>
    - <code>Identify data on which the model needs improvement</code>
</pre><p>and</p><pre class="ql-syntax" spellcheck="false">- You can filter on low inference score, to understand why your model is failing, and how to fix it (getting more data, splitting or merging categories, etc.) 
</pre><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/1051/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>